### PR TITLE
Updates product name in About modal

### PR DESCRIPTION
**Pull request by AlexSCorey**
##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
 - UI

### CODE REVIEW

**Review by nixocio — CHANGES_REQUESTED** [review]
@AlexSCorey I got this code locally, and there is a duplication.

image

File: awx/ui_next/src/components/About/About.jsx [hunk 1]
```diff
@@ -31,7 +36,7 @@ function About({ version, isOpen, onClose }) {
     <AboutModal
       isOpen={isOpen}
       onClose={onClose}
-      productName={brandName.current}
+      productName={`${brandName}`}
```
Comment (keithjgrant) [inline awx/ui_next/src/components/About/About.jsx]: Should this just be `productName={brandName}` ?

**Review by nixocio — APPROVED** [review]
image

**Comment by tiagodread** [comment]
@AlexSCorey could you rebase this one?

**Review by keithjgrant — APPROVED** [review]

**Review by tiagodread — APPROVED** [review]

**Pull request merged**